internal/models/kubernetessecret/cluster/secretexport/secret_export_method_create.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package secret
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportSecretExportRequest Request to create a SecretExport.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.secretexport.CreateSecretExportRequest
+type VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportSecretExportRequest struct {
+
+	// SecretExport to create.
+	SecretExport *VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportSecretExport `json:"secretExport,omitempty"`
```

Comment:
Why do we have 2 `Secretexport`s in the name?

Please check and modify all such occurrences for secrets as well.

Reply:
@Shubbhang351 this is a nit, you can keep it same as well. 

The reason being in the API model we get the full path to the request (auto generated via swagger gen)